src/app/report-rollup/assessment-reports/assessment-reports.component.ts

```diff
@@ -124,4 +128,49 @@ export class AssessmentReportsComponent implements OnInit {
     this.selectedPhastCalcs = selectedCalcs.filter(item => { return item.type == 'furnace' });
     this.selectedFsatCalcs = selectedCalcs.filter(item => { return item.type == 'fan' });
   }
+
+  goToAssessment(assessment: Assessment, mainTab?: string, subTab?: string) {
```

Comment:
tab isn't being used, remove the conditions and assignment and just set the segment to navigate.